modules/routes.js

```diff
@@ -9,12 +9,12 @@ import SimpleForm from './components/SimpleForm'
 
 export default (
   <Route>
-    <Route path="/" component={App}>
-      <IndexRoute component={SimpleForm}/>
+    <Route path='/' component={App}>
+      <IndexRoute component={SimpleForm} />
     </Route>
-    <ServerRoute path="/api">
-      <ServerRoute path=":hello" get={hello}/>
+    <ServerRoute path='/api'>
```

Comment:
True - can remove all the api stuff (though it's pretty cool react-project wise :)